src/packs/ranged/Distortion Cannon.yml

```diff
@@ -1,16 +1,13 @@
 _id: DTz1gyIdoEVpCvZo
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: false
     longReload: false
     max: 6
     suppress: false
     type: typeAPower
     value: 6
-  bundle:
-    amount: 1
-    bundled: false
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.